src/Avalonia.Controls/Presenters/ItemVirtualizer.cs

```diff
@@ -164,10 +185,10 @@ public static ItemVirtualizer Create(ItemsPresenter owner)
             }
 
             var virtualizingPanel = owner.Panel as IVirtualizingPanel;
-            var scrollable = (ILogicalScrollable)owner;
+            var scrollContentPresenter = owner.Parent as IScrollable;
             ItemVirtualizer result = null;
 
-            if (virtualizingPanel != null && scrollable.InvalidateScroll != null)
```

Comment:
`scrollable.InvalidateScroll` was change to an event so we can't check to compare it to `null`, instead just check if its parent is a scroll content presenter (which is effectively what this check does).